rootfs/etc/os-release

```diff
@@ -0,0 +1,9 @@
+NAME="Chimera OS"
+PRETTY_NAME="Chimera"
+ID=chimera
+ID_LIKE=arch
+BUILD_ID=rolling
+ANSI_COLOR="1;31"
+HOME_URL="https://chimeraos.org/"
+DOCUMENTATION_URL="https://chimeraos.org/about"
```

Comment:
I think this should point to the wiki.

Reply:
The wiki is being phased out, so I think for now the about page is okay, or we can omit it.

Reply:
We should have some kind of documentation with common Troubleshooting at least. Maybe not in a GitHub wiki, but in a markdown file somewhere.